docs/packaging-applications/package-repositories/guides/maven-repositories/gitlab-maven-feed.md

```diff
@@ -0,0 +1,33 @@
+---
+title: GitLab Maven repository
+description: Configuring a GitLab Maven repository as an Octopus feed.
+position: 20
+---
+GitLab creates a Maven Registry for each Project or Group.  To add the Maven Registry to Octopus Deploy as an external feed, you will first need to get the Project or Group Id
+
+Project Id
+
+![GitLab Project Id](../images/gitlab-project-id.png)
+
+Group Id
+
+![GitLab Group Id](../images/gitlab-group-id.png)
+
+## Adding a GitLab Maven package feed as an Octopus External Feed
+Create a new Octopus Feed by navigating to **{{Library, External Feeds}}** and select the `NuGet Feed` Feed type. 
+
+Give the feed a name and in the URL field, enter the HTTP/HTTPS URL of the feed for your GitLab Project or Group in the format:
+
+Project:
+https://gitlab.octopusdemos.app/api/v4/projects/4
```

Comment:
```suggestion

```

Reply:
It's not publically accessible and meant for demonstration.  We can alter the server name if that's what you're referring to.